.evergreen/config.yml

```diff
@@ -181,6 +181,19 @@ axes:
           DRIVER_DIRNAME: "ruby"
           DRIVER_REPOSITORY: "https://github.com/mongodb/mongo-ruby-driver"
           DRIVER_REVISION: "master"
+      - id: dotnet-csharp2515
+        display_name: "dotnet (csharp2515)"
+        variables:
+          DRIVER_DIRNAME: "dotnet"
+          DRIVER_REPOSITORY: "https://github.com/vincentkam/mongo-csharp-driver"
+          DRIVER_REVISION: "csharp2515"
```

Comment:
I will change these lines once https://github.com/vincentkam/mongo-csharp-driver/pull/119 is approved and has been merged to master and pushed upstream